src/EditorFeatures/Core/Implementation/Suggestions/PreviewChangesSuggestedAction.cs

```diff
@@ -17,13 +19,23 @@ internal class PreviewChangesSuggestedAction : SuggestedAction
         {
         }
 
-        public override object GetPreview(CancellationToken cancellationToken)
+        public override bool HasPreview
+        {
+            get
+            {
+                // Since PreviewChangesSuggestedAction will always be presented as a
+                // 'flavored' action, it will never havve a preview.
+                return false;
+            }
+        }
+
+        public override Task<object> GetPreviewAsync(CancellationToken cancellationToken)
         {
             // Since PreviewChangesSuggestedAction will always be presented as a
             // 'flavored' action, code in the VS editor / lightbulb layer should
             // never call GetPreview() on it. We override and return null here
             // regardless so that nothing blows up if this ends up getting called.
-            return null;
+            return SpecializedTasks.Default<object>();
```

Comment:
you could add SpecializedTasks.NullObject which hold onto static Task.FromResult<object>(null);